src/test/java/com/github/underscore/math/MathTest.java

```diff
@@ -205,6 +205,48 @@ public void accept(String str) {
         U.chain(new Integer[] {0}).mean();
         U.chain(new Integer[] {0}).median();
         U.chain(new Integer[] {0}).sum();
+        U.chain(new Integer[] {0}).at(new Integer[] {0});
+        U.chain(new String[] {""}).camelCase();
+        U.chain(new String[] {""}).capitalize();
+        U.chain(new String[] {""}).deburr();
+        U.chain(new String[] {""}).drop();
+        U.chain(new String[] {""}).drop(0);
+        U.chain(new String[] {""}).dropRight();
+        U.chain(new String[] {""}).dropRight(0);
+        U.chain(asList(1, 2, 3)).dropRightWhile(new Predicate<Integer>() {
+            public boolean test(Integer n) {
+                return n > 2;
+            }
+        });
+        U.chain(asList(1, 2, 3)).dropWhile(new Predicate<Integer>() {
```

Comment:
\[FindBugs\] INFO: RV: Return value of U$Chain\.dropWhile\(Predicate\) ignored, is this OK in com\.github\.underscore\.math\.MathTest\.chain\(\)